projects/cdk/utils/os/is-apple.ts

```diff
@@ -0,0 +1,16 @@
+import {isIos} from './is-ios';
+
+const SAFARI_REG_EXP = /^((?!chrome|android).)*safari/i;
+
+export function isApple({
+    userAgent,
+    navigator,
+}: {
+    navigator: Navigator | null;
```

Comment:
These things are never `null` and also `userAgent` is a part of the `Navigator` so no need to inject them separately.

Reply:
fixed